src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -81,6 +82,8 @@ function ProposalDetail({
   const { account: connectedAccount } = useWallet()
   const network = getNetwork(chainId)
 
+  const isAbstainProposal = proposal.metadata === 'Abstain proposal'
```

Comment:
Consider put that string on some const `'Abstain proposal'` it used more one time and could avoid some typo. WDYT?

Reply:
Good suggestion.

Reply:
Or make that isAbstainProposal receive proposal and re-use them.